source/extensions/transport_sockets/tls/context_manager_impl.cc

```diff
@@ -47,11 +47,15 @@ ContextManagerImpl::createSslServerContext(Stats::Scope& scope,
   return context;
 }
 
-size_t ContextManagerImpl::daysUntilFirstCertExpires() const {
-  size_t ret = std::numeric_limits<int>::max();
+absl::optional<size_t> ContextManagerImpl::daysUntilFirstCertExpires() const {
+  absl::optional<size_t> ret = absl::make_optional(std::numeric_limits<int>::max());
```

Comment:
Should this use `std::numeric_limits<size_t>::max()`? (I'm not sure if that compiles, but it seems like a type-mismatch here with another implicit cast.) Compare with spiffe_validator.cc where we used `SIZE_MAX`.

Reply:
Sure, it is better.

Reply:
I will unit them as `SIZE_MAX`.